webapp/channels/src/components/admin_console/ldap_wizard/ldap_wizard.scss

```diff
@@ -81,4 +81,57 @@
             line-height: 24px;
         }
     }
+
+    .ldap-expandable-section {
+        margin: 20px 0;
+
+        .ldap-expandable-section-header {
+            display: flex;
+            align-items: center;
+            gap: 8px;
+        }
+
+        .ldap-expandable-section-toggle {
+            padding: 4px 0;
+            border: none;
+            background: none;
+            color: var(--button-bg);
+            font-weight: 600;
+
+            &:hover {
+                text-decoration: underline;
+            }
+
+        }
+
+        .ldap-expandable-arrow {
+            color: var(--button-bg);
+            transition: transform 0.2s ease;
+            
+            &.open {
+                transform: rotate(90deg);
+            }
+        }
+
+        .ldap-expandable-section-content {
+            overflow: hidden;
+            max-height: 0;
+            padding-left: 0;
+            margin-top: 0;
+            opacity: 0;
+            transition: max-height 0.3s ease-out, opacity 0.2s ease-out, margin-top 0.3s ease-out;
+
+            &.expanded {
+                max-height: 800px; // Large enough to contain all settings
+                margin-top: 20px;
+                opacity: 1;
+            }
+        }
+    }
+}
+
+.ldap-help-text-hover-tooltip.tooltipContainer {
```

Comment:
Along the lines of https://github.com/mattermost/mattermost/pull/31190#discussion_r2127444358, this further points to this needing its own component since we're styling these differently. I'm happy to discuss how we might do that if you want

Reply:
Now that the markdown and values are removed, the simple component seems good -- but let me know if you had something else in mind.